doc/i2p.md

```diff
@@ -100,7 +79,35 @@ In general, a node can be run with both onion and I2P hidden services (or
 any/all of IPv4/IPv6/onion/I2P/CJDNS), which can provide a potential fallback if
 one of the networks has issues.
 
-## I2P-related information in Bitcoin Core
+```
+-i2pacceptincoming
+```
+
+Whether your node allows I2P inbound connections (see the next section below).
+
+## Persistent vs transient I2P addresses
+
+The first time Bitcoin Core connects to the I2P router, it automatically
+generates a persistent I2P address and its corresponding private key by default
```

Comment:
Note to self: re-verify if the `-i2pacceptincoming` help ought to be updated regarding "If this is not set but -i2psam is set then only outgoing connections will be made to the I2P network". IIRC listening is the default behavior.

Reply:
Yes it should be updated as its confusing at this moment. Incoming i2p connections work by default if i2psam is set with i2p working correctly. Maybe we could change this to:

```
     Incoming I2P connections are accepted via the SAM proxy
     by default. If this is set to 0 and -i2psam is set
     then only outgoing connections will be made to the I2P network.
     Ignored if -i2psam is not set. Listening for incoming I2P
     connections is done through the SAM proxy, not by binding to a
     local address and port (default: 1)
     
```

Reply:
The text is correct, but I can see how it can be confusing. By "to be set" I meant "to be enabled" or "=1" and "not set" => "disabled" or "=0". Thus the current text:

> If **set** and **-i2psam is also set** then incoming I2P connections are accepted via the SAM proxy. If **this is not set** but **-i2psam is set** then only outgoing connections will be made to the I2P network.

means:

> If **-i2pacceptincomng=1** and **-i2psam=1** then incoming I2P connections are accepted via the SAM proxy. If **-i2pacceptincoming=0** but **-i2psam=1** then only outgoing connections will be made to the I2P network.

I think the text by @1440000bytes is better than the current text. Maybe it can be improved further - to talk about the "default" only at the end. Like "if 5, then it is blue, if 6 then red and if 7 - green. Default 6." rather than "by default it is red, if 5 - blue, if 7 - green. Default 6"